### ISSUE TITLE
Addon listing is missing

### ISSUE THREAD

**Issue by dylanwinn** [ashleybaldock/sixornot#24]
https://addons.mozilla.org/en-GB/firefox/addon/sixornot/ returns 404

**Comment by ashleybaldock** [comment]
Yup. Mozilla de-listed it because of the dependency on knockout.js (which uses unsafe-eval). Needs a UI rewrite using React and I don't currently have time to do that sadly.

**Comment by dylanwinn** [comment]
The addon does still appear to work if you install the xpi manually. Could you upload the xpi somewhere for people that don't want to build it from source?

**Comment by ashleybaldock** [comment]
That is a good idea :) Here - https://morepossibility.com/sixornot-3.3.1.xpi

**Comment by ViBE-HU** [comment]
i just reinstalled one of my machines and noticed this issue. but somehow adding the xpi file is not possible for me. the browser says it's unsigned. how can i add then?

**Comment by ashleybaldock** [comment]
Oh you're right, sorry, I uploaded the xpi produced by my build script which obviously hasn't been signed by Mozilla. Let me fix that.

**Comment by ashleybaldock** [comment]
Ok try this link: [morepossibility.com/sixornot-3.3.0-fx.xpi](https://morepossibility.com/sixornot-3.3.0-fx.xpi) - I have made some progress on a React version just need to find time to finish it :) Hopefully SixOrNot isn't completely dead yet...

**Comment by ashleybaldock** [comment]
(That's the most recent version signed by Mozilla)

**Comment by ViBE-HU** [comment]
it works now. many thanks for your work.
